src/components/Loader.jsx

```diff
@@ -0,0 +1,29 @@
+import Lottie from 'react-lottie'
+import animationData from '../lotties/animation.json'
+
+export const Loader = () => {
+	
+
+	const defaultOptions = {
+		loop: true,
+		autoplay: true,
+		animationData: animationData,
+		rendererSettings: {
+		preserveAspectRatio: 'xMidYMid slice',
+		},
+	}
+
+	return (
+		<div>
+			<Lottie options={defaultOptions} height={300} width={300} />
+		</div>
+	)
+}
+
+//nedan är försök att få animationen att visas längre.
+
+// useEffect(() => {
+// 	export const DelayLottie = setTimeout(() => {
```

Comment:
For the next time you might want to consider creating the function "DelayLottie" before (for example, around line 5) and then you can call the function inside of the useEffect. 👍 
-Izabel&Etna